projects/novo-elements/src/elements/chips/Chips.ts

```diff
@@ -175,15 +175,13 @@ export class NovoChipsElement implements OnInit, ControlValueAccessor {
               this.items.push(value);
             }
           }
-          this._items.next(this.items);
+          this._finalizeItemValue();
```

Comment:
it seems like we want to call `this._finalizeItemValue();` in every scenario or close to every scenario. Could we move this call to the last line of the function (and remove the other calls to it) so it gets call every time? The logic is a little hard to follow with all the if/elses so I understand if that is not possible. I think at least in this part of the function we could move it outside of this if check.

Reply:
thanks for the suggestion! I will look at getting rid of the else blocks and just ending the method with this

Reply:
was able to simplify this a little bit, we do still need to call this new method within the `then` block for the 'noLabels' scenario in order to resolve the issue